### DISCUSSION TITLE
[Question]: Hello, is there any feature like Custome GPTs?

### DISCUSSION THREAD

**Discussion by userbox020** [danny-avila/LibreChat#1448] [Q&A] [answered]
### What is your question?

is there any feature like Custome GPTs?

### More Details

https://openai.com/blog/introducing-gpts

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Yes, it is currently in development, hopefully done before the end of January. It has been discussed here before: https://github.com/danny-avila/LibreChat/discussions/1323

There is also an active discussion about it on discord:
 https://discord.com/channels/1086345563026489514/1172468111023292456
